components/cdc/src/endpoint.rs

```diff
@@ -512,24 +516,26 @@ impl<T: 'static + RaftStoreRouter<RocksEngine>> Endpoint<T> {
         if is_new_delegate {
             // The region has never been registered.
             // Subscribe the change events of the region.
-            let old_id = self.observer.subscribe_region(region_id, delegate.id);
+            let old_id = self
+                .observer
+                .subscribe_region(region_id, delegate.handle.id);
```

Comment:
If assigning `handle.id` to temporary variable, you can save few lines in this file.